tests/tests_port.yml

```diff
@@ -23,9 +29,24 @@
           vars:
             cockpit_packages: minimal
             cockpit_port: 443
+          when: not __bootc_validation | d(false)
```

Comment:
In this case, we are checking that the port customization was set correctly at bootc build time?  So we don't want to run the role?

Do we also have a test that runs the cockpit role against the image built with bootc?

Reply:
> Right -- the goal of these tests is to validate the role run during container build time. We don't currently have tests which run the role against an unconfigured bootc image. They would look pretty much like "standard ostree" tests, but I understand we only have them downstream?

yes, downstream only for now, but I would really like to run the tests upstream, now that it is possible, because of the work you have done.  but that is beyond the scope of this PR

Reply:
Indeed most parts are there -- we could run these same tests with `__bootc_validation` not being defined, then it'll  invoke the role. We need to modify the container build stage to do the .ostree/get-packages dance instead of calling the role, but that shouldn't be too hard.